schematic/manifest/generator.py

```diff
@@ -1657,6 +1675,30 @@ def create_manifests(
 
         Returns:
             Union[List[str], List[pd.DataFrame]]: a list of Googlesheet URLs, a list of pandas dataframes or excel file paths
+
+        ```mermaid
+        sequenceDiagram
+            participant User
+            participant Function
+            participant DataModelParser
+            participant DataModelGraph
+            participant ManifestGenerator
+            User->>Function: call create_manifests
+            Function->>Function: check dataset_ids and validate inputs
+            Function->>DataModelParser: parse data model
+            DataModelParser-->>Function: return parsed data model
+            Function->>DataModelGraph: generate graph
+            DataModelGraph-->>Function: return graph data model
+            alt data_types == "all manifests"
+                Function->>ManifestGenerator: create manifests for all components
+            else
+                loop for each data_type
+                    Function->>ManifestGenerator: create single manifest
+                end
+            end
+            ManifestGenerator-->>Function: return results
+            Function-->>User: return manifests based on output_format
```

Comment:
```suggestion
        sequenceDiagram
            participant User
            participant Function
            participant DataModelParser
            participant DataModelGraph
            participant ManifestGenerator
            User->>Function: call create_manifests
            Function->>Function: check dataset_ids and validate inputs
            Function->>DataModelParser: parse data model
            DataModelParser-->>Function: return parsed data model
            Function->>DataModelGraph: generate graph
            DataModelGraph-->>Function: return graph data model
            alt data_types == "all manifests"
                loop for each component
                    Function->>ManifestGenerator: create manifest for component
                    ManifestGenerator-->>Function: single manifest
                end
            else
                loop for each data_type
                    Function->>ManifestGenerator: create single manifest
                    ManifestGenerator-->>Function: single manifest
                end
            end
            Function-->>User: return manifests based on output_format
```

Reply:
This might not have formatted right. But I updated the bottom section here:

```
            DataModelGraph-->>Function: return graph data model
            alt data_types == "all manifests"
                loop for each component
                    Function->>ManifestGenerator: create manifest for component
                    ManifestGenerator-->>Function: single manifest
                end
            else
                loop for each data_type
                    Function->>ManifestGenerator: create single manifest
                    ManifestGenerator-->>Function: single manifest
                end
            end
            Function-->>User: return manifests based on output_format
```

Entire diagram:
```
        sequenceDiagram
            participant User
            participant Function
            participant DataModelParser
            participant DataModelGraph
            participant ManifestGenerator
            User->>Function: call create_manifests
            Function->>Function: check dataset_ids and validate inputs
            Function->>DataModelParser: parse data model
            DataModelParser-->>Function: return parsed data model
            Function->>DataModelGraph: generate graph
            DataModelGraph-->>Function: return graph data model
            alt data_types == "all manifests"
                loop for each component
                    Function->>ManifestGenerator: create manifest for component
                    ManifestGenerator-->>Function: single manifest
                end
            else
                loop for each data_type
                    Function->>ManifestGenerator: create single manifest
                    ManifestGenerator-->>Function: single manifest
                end
            end
            Function-->>User: return manifests based on output_format
```

Reply:
wow very cool. Thanks Bryan. I didn't know that you could do loop in a mermaid diagram.